src/modules/equipment/controller.js

```diff
@@ -0,0 +1,46 @@
+import db from '../../config/db.js'
+
+const getEquipment = async (req, res) => {
+
+    // const {equipment, facility} = req.query;
+
+    try {
+        console.log(req.query.facility);
+        let filters = '';
+        if (req.query.facility && !req.query.equipment) {
+            filters = `WHERE facility_name LIKE '${req.query.facility}'`
+        } else if (req.query.facility && req.query.equipment) {
+            filters = `WHERE facility_name LIKE '${req.query.facility}' AND equipment_name LIKE '${req.query.equipment}'`
```

Comment:
Don't need to use LIKE for facility_name since it's a select option and not a input text field